ci/scripts/ci_updateversion.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# increment version code, need to be unique to send to store
+./gradlew updateVersionCode -P vCode=$CIRCLE_BUILD_NUM
+
+# increment version on package.json, create tag and commit with changelog
+npm run release -- -m "ci(release): generate **CHANGELOG.md** for version %s"
```

Comment:
I dont like Markdown format in the commit messages

Reply:
Ready remove markdown with plain text